Enable QT_SKIP_AUTO_PLUGIN_INCLUSION and QT_SKIP_AUTO_QML_PLUGIN_INCLUSION by
default
Index: cmake/QtPostProcessHelpers.cmake
--- cmake/QtPostProcessHelpers.cmake.orig
+++ cmake/QtPostProcessHelpers.cmake
@@ -507,7 +507,7 @@ function(qt_internal_create_plugins_auto_inclusion_fil
 
 # Distributions should probably change this default.
 if(NOT DEFINED QT_SKIP_AUTO_QML_PLUGIN_INCLUSION)
-    set(QT_SKIP_AUTO_QML_PLUGIN_INCLUSION OFF)
+    set(QT_SKIP_AUTO_QML_PLUGIN_INCLUSION ON)
 endif()
 
 set(__qt_qml_plugins_config_file_list \"\")
